coretex/cli/commands/project.py

```diff
@@ -20,23 +20,70 @@
 import click
 
 from ..modules import ui, project_utils, utils, user
-from ...entities import Project
-from ...networking import NetworkRequestError
-from ...configuration import loadConfig, saveConfig
+from ...entities import Project, ProjectType, ProjectVisibility
+from ...networking import NetworkRequestError, EntityNotCreated, networkManager, RequestFailedError
+from ...configuration import loadConfig
 
 
-def selectProject(projectId: int) -> None:
+@click.command()
+@click.option("--name", "-n", type = str, help = "Project name")
+@click.option("--type", "-t", type = int, help = "Project type")
+@click.option("--description", "-d", type = str, help = "Project description")
+def create(name: Optional[str], type: Optional[int], description: Optional[str], visibility: Optional[int]) -> None:
+    if name is None:
+        name = ui.clickPrompt("Please enter name of the project you want to create:", type = str)
+
+    if type is None:
+        type = project_utils.selectProjectType()
+
+    if description is None:
+        description = ui.clickPrompt("Please enter your project's description:", type = str)
```

Comment:
Description is optional. Can user create a project without description using this command?